report/report-generator.js

```diff
@@ -102,13 +93,58 @@ class ReportGenerator {
     return '<script src="/pages/scripts/lighthouse-report.js"></script>';
   }
 
+  _createSections(aggregations) {
+    return aggregations.reduce((menu, aggregation) => {
+      if (menu.indexOf(aggregation.type.name) === -1) {
+        menu.push(aggregation.type.name);
+      }
+      return menu;
+    }, []);
+  }
+
+  _createPWAAuditsByTag(aggregations) {
```

Comment:
So this is challenging. The PWA "grouping" contains various aggregations which contain audits. I don't know that we want a mega-aggregation layer here, so for now I've hard-coded them to the report. We can revisit that decision, but it's not necessarily the case that I think we want to be fully dynamic.